src/ocean/text/convert/DateTime_tango.d

```diff
@@ -970,6 +970,41 @@ private struct Result
     }
 }
 
+/*******************************************************************************
+
+    Params:
+        value = time value to wrap in pretty-printing struct
+
+    Returns:
+        wrapper struct which, when supplied to `Formatter`, prints `value`
+        using current default locale settings.
+
+*******************************************************************************/
+
+public AsPrettyStr asPrettyStr ( Time value )
+{
+    return AsPrettyStr(value);
+}
+
+/*******************************************************************************
+
+    Wrapper struct which, when supplied to `Formatter`, prints `value`
+    using current default locale settings.
+
+*******************************************************************************/
+
+public struct AsPrettyStr
+{
+    private Time value;
+
+    public void toString (void delegate(cstring) sink)
+    {
+        // Layout defaults to 'G'
+        scope dg = (cstring s) { sink(s); return s.length; };
+        DateTimeDefault.format(dg, this.value, "");
```

Comment:
Hum since those two differ I think we can just change `format` here... Will look into it

Reply:
What do you mean?

Reply:
This uses a `size_t`-returning delegate and shouldn't. But we can take care of that later.